install/Knative-with-Minikube.md

```diff
@@ -36,7 +36,7 @@ For Linux use:
 minikube start --memory=8192 --cpus=4 \
   --kubernetes-version=v1.11.3 \
   --vm-driver=kvm2 \
-  --bootstrapper=kubeadm \
+  --disk-size=30g \
```

Comment:
How did you come to the number of 30g? What about setting an env variable for this?
```
export DISK_SIZE=30G
```
```
--disk-size=$DISK_SIZE
```

Reply:
I tried with default minikube values for disc size, it broke very badly, so I extended the value by 10 more and it worked, though minishift guide suggests 50g